nailgun/entities.py

```diff
@@ -2056,6 +2056,7 @@ def __init__(self, server_config=None, **kwargs):
                 Repository,
                 required=True,
             ),
+            'size': entity_fields.IntegerField(required=True, min_val=2097152),
```

Comment:
I'm setting `min_val=2097152` because `content_chunk_size = 2 * 1024 * 1024` on L2150. Maybe we should set these to a global variable?

Reply:
I think that would be best, move `content_chunk_size = 2 * 1024 * 1024` to top of the class and `min_val=content_chunk_size`.